fe/fe-core/src/test/java/com/starrocks/connector/hive/HiveMetadataTest.java

```diff
@@ -188,6 +190,15 @@ public void testGetHiveRemoteFiles() throws AnalysisException {
         Assert.assertEquals(2, blockDesc.getReplicaHostIds().length);
     }
 
+    @Test
+    public void testGetFileWithSubdir() throws StarRocksConnectorException {
```

Comment:
it's better to add more cases, like s3://xxxx/xxxx.file or hdfs://xxxx:10010/ccc/ddd. you colud only test the 'getSuffixName()'